capstone/src/main/webapp/index.html

```diff
@@ -3,13 +3,17 @@
   <head>
     <title>UniViz Main Page</title>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <h2>Welcome to UniViz!</h2>
     <form autocomplete="off">
       <label for="search">Search for colleges:</label><br>
-      <input type="text" id="search" name="search"><br>
+      <input type="text" id="search" name="search" placeholder="College Name" oninput="giveSuggestions()" onkeydown="keyDown(event)"><br>
       <input type="submit" value="See your colleges!">
     </form>
+    <p>Current List of Suggested Colleges:</p>
+    <ul id="suggestions"></ul>
```

Comment:
First i thought this is the list item that is showing the suggestions. But after reading the code i realized this is the list of colleges that the user selected. So a better name / text would help. Something like "Current list of selected colleges". Same with element id.

Reply:
done! updated the label